Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM node:9-alpine
+
+RUN npm i npm@latest -g
+
+## Install build toolchain, install node deps and compile native add-ons
+RUN apk add --no-cache --virtual python make g++
+
+# Create app directory
+RUN mkdir -p /home/node/app/node_modules && chown -R node:node /home/node/app
+
+# Set the working directory for the application
+WORKDIR /home/node/app
+
+# Before running install, switch to non-root user "node"
+USER node
+
+# A wildcard is used to ensure both package.json AND package-lock.json are copied
+COPY package*.json ./
+
+# Install app dependencies
+RUN npm ci
```

Comment:
Why are we running `npm ci` here rather than just `npm i`?

Reply:
By using `npm ci` it prevents changes being made to the `package.json` or `package-lock.json` files, and it will ensure that every launch is consistent. From the npm docs, it is also faster than running just `npm i`

Reply:
Ok yeah that makes sense. We probably don't want to update the dependencies when deploying to production.